config/more_samples-glyf_colr_1.py

```diff
@@ -683,6 +683,85 @@ def _composite(composite_mode, accessor_char):
     )
 
 
+def _foreground_color(fill_type, accessor_char):
+
+    fill_type_map = {
+        "solid": {
+            "Format": ot.PaintFormat.PaintSolid,
+            "PaletteIndex": 0xFFFF,
+            "Alpha": 1,
```

Comment:
I wonder what happens or should happen when alpha is not 1? Do you get the user-defined foreground color with some transparency applied? I see all the 0xFFFF here have alpha=1, maybe you can add one where it != 1 if this is something we support.

Reply:
Added 0.3 alpha cases, that's supported, yes. Changed the secondary color, too, for better inspection.

![image](https://user-images.githubusercontent.com/1021784/140939322-3b0ebd73-b7f3-400d-bdac-b48e63e64ed9.png)

Reply:
thanks!